ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -290,6 +290,12 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp-upi-xpn
+- as: e2e-gcp-ovn-xpn
+  optional: true
+  run_if_changed: ^(ipi/gcp/.*)
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
```

Comment:
Any reason you're using this cluster_profile rather than `gcp`? I would expect this uses a different set of credentials than `gcp`.